gremlin-go/driver/driverRemoteConnection.go

```diff
@@ -176,6 +183,18 @@ func (driver *DriverRemoteConnection) GetSessionId() string {
 	return driver.client.session
 }
 
+func (driver *DriverRemoteConnection) commit() (ResultSet, error) {
+	bc := &bytecode{}
+	bc.addSource("tx", "commit")
+	return driver.submitBytecode(bc)
+}
+
+func (driver *DriverRemoteConnection) rollback() (ResultSet, error) {
+	bc := &bytecode{}
+	bc.addSource("tx", "rollback")
+	return driver.submitBytecode(bc)
+}
```

Comment:
@lyndonbauto should these 2 methods return `ResultSet` or just omit it?

Reply:
Discussed off of here